client/src/components/TimetableSwitcher/TimetableSwitcher.tsx

```diff
@@ -58,6 +64,25 @@ const TimetableSwitcher = memo(() => {
 		else setAction(inputAction);
 	};
 
+    const handleSaveTimetable = async () => {
+        const input = document.getElementById('timetable') as HTMLElement;
+        const imgData = await domtoimage.toPng(input)
+        triggerBase64Download(imgData, 'FFCSThingy - Timetable');
```

Comment:
Change the name to reflect active timetable please.
```suggestion
        triggerBase64Download(imgData, `FFCSThingy - ${activeTimetableName}`);
```